Tests/Schema.NET.Test/Examples/CourseTest.cs

```diff
@@ -10,7 +10,7 @@ public class CourseTest
     {
         Name = "Introduction to Computer Science and Programming", // Required
         Description = "Introductory CS course laying out the basics.", // Required
-        Provider = new Organization() // Recommended
+        SourceOrganization = new Organization() // Note: This is typically set via "Provider" (a pending property) and is required
```

Comment:
The `Provider` property is marked as pending now so isn't available in our tests (which use the `Schema.NET`, not `Schema.NET.Pending`